### PR TITLE
sanitize some unescaped HTML from CDP

### PR DESCRIPTION
**Pull request by paulirish**
fixing this: 
image

see the actual fixes this makes in the generated HTML: https://gist.github.com/paulirish/f7f86e982487a23f494b6c834841743f

We should fix a bunch of these in the PDL and write some validation there so new raw HTML can't be introduced without backticks.

R=@thiagowfx
cc @OrKoN

### CODE REVIEW

File: pages/domainGenerator.js [hunk 1]
```diff
@@ -15,6 +16,18 @@ const itemSort = (a, b) => {
...
 };
 
+// This handles a few weird cases of raw HTML or over-escaping in the protocol JSON
+function parseSafeMarkdown(mdText) {
+  mdText = mdText.replaceAll('<', '&lt;'); // protect against some naked `<script>` and `<label>` and `<input` in the JSON
+  const html = marked(mdText);
+  return html
+    .replaceAll('&amp;lt;', '&lt;')  // Page.printToPDF > headerTemplate.... and IO.StreamHandle
+    .replaceAll('&amp;gt;', '&gt;')  //  IO.StreamHandle
+    .replaceAll('&amp;gt', '&gt;')  //  IO.StreamHandle (actual typo in desc)
```
Comment (thiagowfx) [inline pages/domainGenerator.js:26]: Better fix the typo right at the source then.

File: pages/domainGenerator.js [hunk 2]
```diff
@@ -15,6 +16,18 @@ const itemSort = (a, b) => {
   return (a.name || a.id).localeCompare(b.name || b.id);
 };
 
+// This handles a few weird cases of raw HTML or over-escaping in the protocol JSON
+function parseSafeMarkdown(mdText) {
+  mdText = mdText.replaceAll('<', '&lt;'); // protect against some naked `<script>` and `<label>` and `<input` in the JSON
+  const html = marked(mdText);
```
Comment (thiagowfx) [inline pages/domainGenerator.js:22]:
```suggestion
  mdText = mdText.replaceAll('<', '&lt;'); // protect against some naked `<script>` and `<label>` and `<input>` in the JSON
  const html = marked(mdText);
```

File: pages/domainGenerator.js [hunk 3]
```diff
@@ -15,6 +16,18 @@ const itemSort = (a, b) => {
...
+  return html
+    .replaceAll('&amp;lt;', '&lt;')  // Page.printToPDF > headerTemplate.... and IO.StreamHandle
+    .replaceAll('&amp;gt;', '&gt;')  //  IO.StreamHandle
+    .replaceAll('&amp;gt', '&gt;')  //  IO.StreamHandle (actual typo in desc)
+    .replaceAll('&lt;p&gt;Note that', '<p>Note that').replaceAll('their child nodes.&lt;/p&gt;</p>', 'their child nodes.</p></p>') // DOM domain description
```
Comment (thiagowfx) [inline pages/domainGenerator.js:27]: This one also looks odd. Better fix at the source

**Review by thiagowfx — CHANGES_REQUESTED** [review]

File: pages/domainGenerator.js (hunk 1 above)
Comment (paulirish) [reply pages/domainGenerator.js:26]: yeah this one was extra dumb.

well.. i fixed allllllll things at the source ... 
https://chromium-review.googlesource.com/c/chromium/src/+/4704893

after it lands ill revise this PR to be more minimal, probably

Comment (thiagowfx) [reply pages/domainGenerator.js:26]: Sounds good

Comment (paulirish) [reply pages/domainGenerator.js]: updated. ptal

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**